src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html

```diff
@@ -0,0 +1,158 @@
+<template>
+  <pcard if.bind="viewMode === 'edit'">
+    <pform-input
+      helper-message.bind="tokenInfoLoading ? 'Searching token address details...': ''"
+      input-reference.bind="addressInput"
+      label="Token Address"
+      label-info="Some info here"
+    >
+      <pinput-text
+        input.trigger="getTokenInfo(token.address)"
+        placeholder="0x..."
+        validation-state.bind="tokenInfoLoading? 'validating': addressInput.validationState !== 'validating' ? addressInput.validationState : undefined"
+        value.bind="token.address & validate"
+        view-model.ref="addressInput"
+      ></pinput-text>
+    </pform-input>
+    <div if.bind="infoSearched">
```

Comment:
```suggestion
    <div if.bind="showTokenDetails">
```
For me
`infoSearched` - is what the code is doing
`showTokenDetails` - what the variable is about

Reply:
removed the div